tenseal/cpp/context/tensealencoder.h

```diff
@@ -115,7 +115,7 @@ class TenSEALEncoder {
     /*
     Stores a shared_pointer to the SEAL Context.
     */
-    std::shared_ptr<SEALContext> _context;
+    SEALContext _context;
```

Comment:
So we now keep a full copy of the context instead of a pointer? Why is that?

Reply:
They dropped the Create call, which created the shared_ptr, and they are copying the SealContexts everytime.

Actually, I kept the shared_ptr in the TenSEAL context, we can do that here as well.

Reply:
Yeah, I guess it doesn't require a full copy, as we will be mainly accessing attributes, or do you have a reason to believe we shouldn't use a pointer here?

Reply:
I will revert it here. Initially, I was going to drop std::shared_ptr<SEALContext> everywhere, and this is some partial change.